README.md

```diff
@@ -24,3 +24,83 @@ As time permits, we will continue our efforts to add documentation, guidelines,
 <area shape="circle" coords="384,704,52" alt="User Acceptance Testing " target="_blank" href="https://www.toolsqa.com/software-testing/user-acceptance-testing-uat/" />
 <area shape="circle" coords="383,868,65" alt="SDLC - Software Development Life Cycle" target="_blank" href="https://www.softwaretestingmaterial.com/sdlc-software-development-life-cycle/" />
 </map>
+
+
+### DevOps
```

Comment:
Nice! I like how you split the images up into individual links since Github doesn't allow SVG mapping. 👍